### ISSUE TITLE
Could someone please explain this ? 

### ISSUE THREAD

**Issue by complete-dope** [ABaldrati/CLIP4Cir#19]
logits = combiner(reference_image_features, text_features, target_image_features)
ground_truth = torch.arange(images_in_batch, dtype=torch.long, device=device)
loss = crossentropy_criterion(logits, ground_truth)

why are the ground truths calculated like this and how are they ground truth ?  @ABaldrati 

file: combiner_train.py

**Comment by duong-bq** [comment]
Me too, have you found the answer yet?
